### PR TITLE
Рябов -- Лабораторная работа #1

### PR DESCRIPTION
**Pull request by Esth3ro**

### CODE REVIEW

**Comment by Esth3ro** [comment]
@Fznamznon

**Comment by Esth3ro** [comment]
@dimasique1

File: code/ryabov-avl-tree/src/test/java/ru/unn/agile/AVL/Model/AVLTreeTest.java [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+    }
+
+    @Test
+    public void findNull() {
+        AVLTree<Integer, Integer> tree = new AVLTree<>();
+        assertEquals(tree.find(null), null);
+    }
+
+    @Test
+    public void insertAndFindInSequence() {
```
Comment (Fznamznon) [inline code/ryabov-avl-tree/src/test/java/ru/unn/agile/AVL/Model/AVLTreeTest.java]: Please add more test cases to check three balancing. You can try test with adding reverse sorted sequence (eg 5 6 4 3 2 1).

File: code/ryabov-avl-tree/src/main/java/ru/unn/agile/AVL/Model/AVLTree.java [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+                current = current.getRight();
+            } else {
+                return current;
+            }
+        }
+
+        return null;
+    }
+
+    private int height(final Node<K, V> node) {
```
Comment (Fznamznon) [inline code/ryabov-avl-tree/src/main/java/ru/unn/agile/AVL/Model/AVLTree.java]: I suggest rename height --> getHeight.

**Comment by dimasique1** [comment]
Looks good to me after @Fznamznon comments are fixed

**Comment by Fznamznon** [comment]
👍 LGTM

**Comment by Esth3ro** [comment]
@ValManP

**Pull request merged**